hilti/CMakeLists.txt

```diff
@@ -9,3 +9,23 @@ if ( HAVE_TOOLCHAIN )
 else ()
     add_custom_target(hilti-tests DEPENDS hilti-rt-tests hilti-rt-configuration-tests)
 endif ()
+
+option("HILTI_DEV_PRECOMPILE_HEADERS" "Precompile headers for developer tests" ON)
+
+if (${HILTI_DEV_PRECOMPILE_HEADERS} AND TARGET hilti-config AND NOT USE_SANITIZERS)
```

Comment:
Suggest to not do the `USE_SANITIZERS` check here, but instead unset `HILTI_DEV_PRECOMPILE_HEADERS` in the `if ( USE_SANITIZERS )` block in the top-level `CMakeLists.txt.` It'll be easier to find there in the future and it'll show how in the configure output that it was disabled.

Reply:
And a question: what does `AND TARGET hilti-config` do?